package.json

```diff
@@ -16,8 +16,8 @@
     "serve": "gatsby serve",
     "clean": "gatsby clean",
     "typecheck": "tsc --noEmit",
-    "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook",
+    "storybook": "npm run download-data && start-storybook -p 6006",
```

Comment:
Why are we adding back these commands? We decided to remove them

Reply:
fixed